src/AutoRest.CSharp/Common/Output/Models/Serialization/Json/JsonAdditionalPropertiesSerialization.cs

```diff
@@ -6,16 +6,13 @@
 
 namespace AutoRest.CSharp.Output.Models.Serialization.Json
 {
-    internal class JsonAdditionalPropertiesSerialization
+    internal class JsonAdditionalPropertiesSerialization : JsonPropertySerialization
```

Comment:
Just a minor issue: should this be renamed as `JsonAdditionalPropertySerialization`?